hudi-common/src/main/java/org/apache/hudi/common/fs/ConsistencyGuardConfig.java

```diff
@@ -36,15 +36,15 @@
   // time between successive attempts to ensure written data's metadata is consistent on storage
   private static final String INITIAL_CONSISTENCY_CHECK_INTERVAL_MS_PROP =
       "hoodie.consistency.check.initial_interval_ms";
-  private static long DEFAULT_INITIAL_CONSISTENCY_CHECK_INTERVAL_MS = 2000L;
+  private static long DEFAULT_INITIAL_CONSISTENCY_CHECK_INTERVAL_MS = 400L;
```

Comment:
I have also fixed some default configs. If incase someone is using FailSafeConsistencyGuard, in case of data file not present, this guard will execute for 256 secs which may not be required. Hence curtailing to 140 secs max.

Reply:
Sounds good.